analyzers/src/SonarAnalyzer.CSharp/Rules/ReturnEmptyCollectionInsteadOfNull.cs

```diff
@@ -127,27 +127,44 @@ private static BlockSyntax GetBody(SyntaxNode node)
                 case SyntaxKindEx.LocalFunctionStatement:
                     return ((LocalFunctionStatementSyntaxWrapper)node).Body;
 
+                case SyntaxKind.OperatorDeclaration:
+                    return ((OperatorDeclarationSyntax)node).Body;
+
                 default:
                     return null;
             }
         }
 
-        private static LiteralExpressionSyntax GetNullLiteralOrDefault(ArrowExpressionClauseSyntax expressionBody)
-        {
-            var innerExpression = expressionBody.Expression.RemoveParentheses();
-            return innerExpression.IsNullLiteral()
-                ? (LiteralExpressionSyntax)innerExpression
-                : null;
-        }
-
-        private static IEnumerable<ReturnStatementSyntax> GetReturnNullStatements(BlockSyntax methodBlock)
-        {
-            return methodBlock.DescendantNodes(n =>
+        private static IEnumerable<SyntaxNode> GetReturnNullOrDefaultExpressions(SyntaxNode methodBlock) =>
+            methodBlock.DescendantNodes(n =>
                     !n.IsAnyKind(SyntaxKindEx.LocalFunctionStatement,
```

Comment:
`SyntaxKindEx.LocalFunctionStatement` can be moved to a new line as the rest (or put every parameter in line)